### PR TITLE
Add sponsors

### PR DESCRIPTION
**Pull request by radar**
As per @jocranford's comments in Slack last week:

> they need to go in order of sponsorship package actually @ryanbigg - Ruby first (Envato + REA), Emerald second (Culture Amp + Lookahead), Sapphire (none yet) then Opal (One flare)

So we have Ruby (Envato + REA):

image

Then Emerald (Culture Amp + Lookahead):

image

Then Opal (OneFlare):

image

### CODE REVIEW

File: app/views/pages/sponsorship.html.erb [hunk 1]
```diff
@@ -15,3 +15,25 @@
...
+    <%= image_tag "sponsors/culture-amp.png", class: "sponsor-logo-emerald" %>
+    <%= image_tag "sponsors/lookahead.jpg", class: "sponsor-logo-emerald" %>
+  </section>
+
+  <section class='sponsor-logos'>
+    <h2>Opal Sponsors</h2>
+    <%= image_tag "sponsors/oneflare.png", class: "sponsor-logo-opal" %>
+  </section>
+
+  they need to go in order of sponsorship package actually @ryanbigg - Ruby first (Envato + REA), Emerald second (Culture Amp + Lookahead), Sapphire (none yet) then Opal (One flare)
```
Comment (jocranford) [inline app/views/pages/sponsorship.html.erb]: Should this be here??

**Comment by jocranford** [comment]
Should we manually size the images instead of using CSS? 

So far all of the logos have a similar height / width ratio but we'll run into issues if we get some that don't - Cogent for example: http://rubyconf.org.au/2016/sponsors

File: app/views/pages/sponsorship.html.erb (hunk 1 above)
Comment (radar) [inline app/views/pages/sponsorship.html.erb]: Oops, no. I hadn't pushed the commit to remove those.

**Comment by radar** [comment]
I have so far manually resized all the images to 500x150px. Then the CSS is used to resize them down to the appropriate sizes for the sponsorship levels. This way, if a company changes sponsorship levels we can change the CSS class rather than changing the logo.

I'll put a README in the sponsors logo directory which covers that.

**Pull request merged**